essim/inc/internal.h

```diff
@@ -15,14 +15,62 @@
 #include <essim/essim.h>
 
 #include <stdint.h>
+#include <time.h>
+#include <stdio.h>
+#include <assert.h>
+#include <math.h>
 
 #if defined(__cplusplus)
 extern "C" {
 #endif /* defined(__cplusplus) */
 
 enum { LOG2_ALIGN = 6, ALIGN = 1 << LOG2_ALIGN };
 
+#define INTR_PROFILING_PRINTS 0
+#define ENABLE_ONLY_C_PATH 0
+#define ARM_BUILD_FIXES 1
+#define NEW_SIMD_FUNC 1
+
+#define ENABLE_MINK_3 0 //Default mink_4 is choosen
+
+#if ENABLE_MINK_3
+#define SSIM_POOLING_MINKOWSKI_P 3
```

Comment:
as we discussed let's change this to support both Mink 3 and Mink 4 pooling as runtime parameters so we can experiment with which one correlates better with subjective scores.

Reply:
Runtime configurability added.